beacon_chain/utils/shuffling/src/specs/shuffle_test_vectors.yaml

```diff
@@ -0,0 +1,169 @@
+# This file was generated with a modified version of shuffling_sandbox
+# python3 sandbox.py test_vectors
+#diff --git a/sandbox.py b/sandbox.py
+#index 99b0ba7..cd7cafe 100644
```

Comment:
10 points for getting a commit hash that ends in "cafe" :rofl:

Reply:
Thanks for pointing this out 😆 .
But not quite at the end 😩 
```
0 git diff-tree head
18d6e1af6022a277b3b2d7b004b25119df91ba4d
:100644 100644 99b0ba736a0a70594e15e522af0209b240be7e1b cd7cafeedb674a059155931a798cb6796e5b6014 M	sandbox.py
```